docs/book/helpers/head-title.md

```diff
@@ -71,45 +72,49 @@ echo $this->headTitle(); // <title>My albumsMusic</title>
 To explicitly append content, the second paramater `$setType` or the concrete
 method `append()` of the helper can be used:
 
-=== "Invoke Usage"
-    ```php
-    $this->headTitle('My albums')
-    $this->headTitle('Music', 'APPEND');
-    
-    echo $this->headTitle(); // <title>My albumsMusic</title>
-    ```
-
-=== "Setter Usage"
-    ```php
-    $this->headTitle('My albums')
-    $this->headTitle()->append('Music');
-    
-    echo $this->headTitle(); // <title>My albumsMusic</title>
-    ```
+#### Invoke Usage
+
+```php
+$this->headTitle('My albums');
+$this->headTitle('Music', 'APPEND');
+
+echo $this->headTitle(); // <title>My albumsMusic</title>
+```
+
+#### Setter Usage
+
+```php
+$this->headTitle('My albums');
+$this->headTitle()->append('Music');
+
+echo $this->headTitle(); // <title>My albumsMusic</title>
+```
```

Comment:
This is wrong because this removes the tabs. See: https://docs.laminas.dev/laminas-view/helpers/head-title/#append-content

Reply:
@gsteel In cases like this, you can use markdown-lint comments to disable checks for specific rules.